src/app/sidebar/sidebar.component.css

```diff
@@ -0,0 +1,52 @@
+.sidebar {
```

Comment:
Smart decision to put sidebar and other components into their own folder. Good organization 👍